corpus/templates/pages/sig.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block title %}
+{{sig.name}}
+{% endblock %}
+
+{% block content %}
+<section
+    class="hero min-h-[50vh] min-w-screen bg-gradient-to-br from-sky-50 to-blue-300 dark:from-blue-950 dark:to-base-100">
+    <div id="particles-js" class="h-[50vh] w-full"></div>
+    <div style="gap: 20px; justify-content: space-between; margin: auto;">
```

Comment:
Can you use flexbox here? The images (2 or 3) should be horizontal on larger screens, and vertical (and smaller) on smaller screens.

Reply:
Sure done